app/shared/components/Producers/BlockProducers.js

```diff
@@ -96,7 +97,9 @@ class BlockProducers extends Component<Props> {
     } = actions;
     if (validate.NODE) {
       getProducers();
-      getProducersInfo();
+      if (connection.supportedContracts.includes('producerinfo')) {
```

Comment:
If you're going to deeply inspect properties like this, I'd recommend checking if both connection and connection.supportedContracts exists (or ensure that same check is being performed before this is loaded).

If you don't, it'll crash the UI if either is undefined.

Reply:
Yeah that's a good idea!  To be on the safe side.